django_dbq/models.py

```diff
@@ -58,7 +58,9 @@ class Meta:
     objects = JobManager()
 
     def save(self, *args, **kwargs):
-        if not self.pk:
+        is_new = not Job.objects.filter(pk=self.pk).exists()
```

Comment:
If anyone can think of a better way of doing this, please let me know. With Django's built-in `UUIDField`, `self.pk` is already a UUID at this point.